.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
```

Comment:
I’m not really a fan of having editor specific configurations within a repository, different people may want them setup differently. @Jason2605 what’s your thoughts?

Reply:
This config file currently specifically contains just the launch profiles for easily attaching a debugger without needing to configure it to attach onto the docker container or locally running process. I'd say it's worth adding this to make debugging easy for vs-code users.